src/ceph-disk

```diff
@@ -1373,13 +1958,26 @@ def prepare_journal_dev(
                     ),
                 '--mbrtogpt',
                 '--',
-                journal,
+                journal_dev,
                 ]
             )
 
         update_partition(journal, 'prepared')
 
+        # wait for udev event queue to clear
+        udev_settle()
+
         LOG.debug('Journal is GPT partition %s', journal_symlink)
+        #
```

Comment:
This part needed some fixup, the wait partition is followed by getting a new device context for the  journal partition and then getting the journal symlink from the device context (based on the device) For multipath device we had preferred to go with journal symlink to the /dev/mapper/<partition> as the by-partuuid links kept getting reset to the underlying raw device. If you think this is not needed, I will revert the logic here